xbmc/guilib/GUISliderControl.cpp

```diff
@@ -614,7 +614,7 @@ void CGUISliderControl::SetFromPosition(const CPoint &point, bool guessSelector
   {
     case SLIDER_CONTROL_TYPE_FLOAT:
     {
-      float fValue = m_fStart + (m_fEnd - m_fStart) * fPercent + m_fInterval / 2;
+      float fValue = m_fStart + (m_fEnd - m_fStart) * fPercent;
```

Comment:
I can't really tell if what's being done here is right sorry.
Either way, isn't it a bit risky for nexus at this point?

Reply:
yes is not immediate to understand,
i have made this change time ago:
https://github.com/xbmc/xbmc/commit/ac3f7c94be557682659fe06591bf10d7b7bd4f72
to fix slider value when you move with mouse/touch the slider circle

i removed it because i have changed RoundF method,
you can see the difference between the old/new here:
https://godbolt.org/z/5Yv19E7G9
take a look to the last value (9.99100208282470703125) of both RoundF methods,
last value should be 10, but the old RoundF was returning less than 10 because was "rounding near to 0",
then to fix the rounding i had add `+ m_fInterval / 2`

with the new RoundF  now round "near to the value" then `+ m_fInterval / 2` can be removed
in any case rounding to a multiple cannot be always perfect
but for the GUI control `fValue` goto SetFloatValue that has a double check then cannot make problems
there are no risks

Reply:
oh, disregard. This seems to be only used by the slider control (and introduced by you anyway).